apps/labs/posts/automated-accessibility-testing-with-bokeh.md

```diff
@@ -0,0 +1,178 @@
+---
+title: 'Automated Accessibility Testing with Bokeh'
+authors: [chiemezuo-akujobi]
+published: September 10, 2024
+description: 'Data is only useful to the extent that people can access it.
+  Let me walk you through one of several steps toward making Bokeh data
+  visualizations more accessible.'
+category: [Access-centered, Beyond PyData]
+featuredImage:
+  src: /posts/hello-world-post/featured.png
+  alt: 'Excellent alt-text describing the featured image which appears as a
+    small image in our main blog page. This image needs to be 391x174 pixels
+    in size.'
+hero:
+  imageSrc: /posts/hello-world-post/hero.jpeg
+  imageAlt: 'Excellent alt-text describing the hero image which appears as a
+    large image at the top of your post page. This image needs to be 1440x696
+    pixels in size'
+---
+
+My name is Chiemezuo, and I got to do some amazing work as a Quansight Labs
+intern this summer. My project was on developing Automated Accessibility Testing
+for the [Bokeh](https://bokeh.org/) project, and I'm here to tell you all about
+it. So, fasten your seatbelts for a fun ride. The best part about this story is
+that neither my mentor nor I had previously worked with the codebase prior to
+this internship.
```

Comment:
Let's brainstorm a way to hook the reader into your piece. The bit about how neither of us had worked with the codebase is exactly the kind of thing I'm thinking about because I think it might make the reader intrigued. But I think we can come up with something even more compelling.